voter-dapp/src/ActiveRequests.js

```diff
@@ -374,6 +435,44 @@ function ActiveRequests({ votingAccount, votingGateway }) {
     }
   };
 
+  const copyStringToClipboard = string => {
+    // Source for implementation details: https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript
+    if (!navigator.clipboard) {
+      // Synchronously copy
+      const textArea = document.createElement("textarea");
+      textArea.value = string;
+
+      // Avoid scrolling to bottom
+      textArea.style.top = "0";
+      textArea.style.left = "0";
+      textArea.style.position = "fixed";
+
+      document.body.appendChild(textArea);
+      textArea.focus();
+      textArea.select();
+
+      try {
+        const successful = document.execCommand("copy");
+        const msg = successful ? "successful" : "unsuccessful";
+        console.log("Sync: Copying text command was " + msg);
+      } catch (err) {
+        console.error("Sync: Could not copy text: ", err);
+      }
+
+      document.body.removeChild(textArea);
+    }
+
+    // Asynchronously copy. This has been the way to copy to the clipboard for Chrome since v66.
+    navigator.clipboard.writeText(string).then(
+      function() {
+        console.log(`Async: Copied to clipboard`);
+      },
+      function(err) {
+        console.error("Async: Could not copy text: ", err);
+      }
```

Comment:
Should this be in an `else`? I assumed that this code should only be run if we don't use the other method for copying, but maybe I'm misunderstanding?

Reply:
yeah you're right. Either synchronously copy or async copy